stbt/grid.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2018-2019 Stb-tester.com Ltd.
+
+"""What is a grid?
+
+Appearance
+==========
+
+Regularly spaced elements.
+
+Variable number of items.
+
+Laid out in order.
+
+Empty cells at the end, but no empty cells in the middle.
```

Comment:
I don't think this comment makes sense for the implementation below.  Probably should be stripped out.

Reply:
Done.